docs/en/userguide/user-documentation.md

```diff
@@ -0,0 +1,97 @@
+# CKAN Registry
+
+The CKAN Registry module adds a new page type to the CMS, allowing a content editor to add pages that display data coming from an external CKAN resource in a table format to website visitors.
+
+> For the purpose of this help text we will reference [data.govt.nz](https://catalogue.data.govt.nz) as our CKAN instance. This could easily be any CKAN instance that has API access enabled.
+
+After creating a new CKAN Registry Page, the normal page editing options are available to the content editor, along with a new tab called "Data". This tab will allow one to paste a CKAN datastore resource URL into the "Data source URL" field.
```

Comment:
I'm actually finding it pretty hard to understand what we should do from that guide. The text above the table says to use bold for sections like Security or Pages, but to use quotes for UI elements "e.g. Buttons or Tabs".

The examples in the table are a mixture of both.

I'll update all tab names to be bold and italic, and all field labels to be bold and quoted, under the assumption that you'll update the Confluence page to reflect this rule =)

Reply:
Our guide seems to be incorrect, thanks for the heads up. It's been a work in progress since we started updating user help. I'll update the guide but for future reference: 
* Buttons and tabs are bold and italicised.
* Fields and pages are bold. 
After I've reviewed I can go through and push those styling changes if you'd like.